modules/appeals_api/app/services/appeals_api/remove_pii.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+module AppealsApi
+  class RemovePii
+    include SentryLogging
+
+    APPEALS_TYPES = [
+      HigherLevelReview,
+      NoticeOfDisagreement
+    ].freeze
+
+    def initialize(form_type:)
+      @form_type = form_type
+    end
+
+    def run!
+      validate_form_type!
+
+      result = remove_pii!
+
+      log_failure_to_sentry if records_were_not_cleared(result)
+
+      result
+    end
+
+    private
+
+    attr_accessor :form_type
+
+    def remove_pii!
+      records_to_be_expunged.update(form_data: nil, auth_headers: nil)
+    end
+
+    def validate_form_type!
+      raise ArgumentError, 'Invalid Form Type' unless valid_form_type
+    end
+
+    def valid_form_type
```

Comment:
Not sure this method is even necessary since ruby already gives us `in?`, and it's only used in one spot. That or it should be a predicate method since it's a question being asked of the data.

Reply:
I agree it should be a predicate method! I think the `valid_form_type?` wrapper of the `in?` method is for readability. Is that what you're referring to?